weldx/transformations/local_cs.py

```diff
@@ -547,6 +547,11 @@ def is_time_dependent(self) -> bool:
         """
         return self.time is not None or self._coord_ts is not None
 
+    @property
+    def has_timeseries(self) -> bool:
+        """Return `True` if the coordinate system has a `TimeSeries` component."""
+        return self._coord_ts is not None
```

Comment:
```suggestion
        return isinstance(self.coordinates, TimeSeries) or isinstance(self.orientation, TimeSeries)
```
I guess this would make sense?